thirdparty/stb/stb_vorbis.c

```diff
@@ -2300,7 +2300,14 @@ void inverse_mdct_slow(float *buffer, int n, vorb *f, int blocktype)
    int i,j;
    int n2 = n >> 1, nmask = (n << 2) -1;
    float *x = (float *) malloc(sizeof(*x) * n2);
-   memcpy(x, buffer, sizeof(*x) * n2);
+   if (x == NULL) {
+       // handle error
+   }
+   if (sizeof(*x) * n2 <= sizeof(buffer)) {
```

Comment:
Isn't `sizeof(buffer) == sizeof(float*) == sizeof(void*) == 8`? In that case this doesn't do the desired thing.
Also note that this change won't have any effect anyway, because it's in code that's behind a `#elif 0` so will never be compiled.